content/blog/2023/05/camel-debezium-quarkus/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Change Data Capture with Apache Camel and Debezium"
+date: 
+draft: true
+authors: [mikeintoch]
+categories: ["Features", "Camel"]
+preview: "Capturing database change records using Apache Camel and Debezium"
+---
+<p align="center">
+<img src="header-image.svg" width="479" height="244">
+</p>
+
+In this article, we will explore how to use Debezium and Apache Camel in conjunction with Quarkus to build a reactive application efficiently and send real-time changes from a database to a destination, such as webservice or message queue or another database.
+
+# What is Debezium?
+
+Debezium is a distributed, event-driven data change platform used to capture data changes in database systems and send them to other systems in real time so that applications can respond to all database insertions, updates and deletions made by other applications. Debezium can connect to a variety of database systems, including MySQL, PostgreSQL, MongoDB, SQL Server and Oracle.
+
+# What is Apache Camel?
+
+Apache Camel is an open source integration framework that provides a wide variety of components and integration patterns to help developers integrate applications efficiently. Camel integrates easily with other systems and technologies, making it a valuable tool for developers who need to integrate multiple systems efficiently.
+
+# What is Quarkus?
+
+Quarkus is a Java application development framework designed for the cloud and optimized for OpenJDK HotSpot and GraalVM. Quarkus is based on the Java EE platform and provides a complete set of features for developing cloud-native applications. Quarkus also supports several popular open source frameworks and libraries, making it easy to integrate technologies such as Debezium and Apache Camel.
```

Comment:
Are you sure about `Quarkus is based on the Java EE platform`? I may be wrong but for me, it was more Eclipse Microprofile, have a look to https://developers.redhat.com/blog/2020/09/11/quarkus-and-jakarta-ee-together-or-not

Reply:
Change text to update Quarkus definition